tests/ArbitraryGeneratedTestTypes.hs

```diff
@@ -15,10 +14,6 @@ import qualified TestProtoImport
 import qualified TestProtoOneof
 import qualified TestProtoOneofImport
 
-instance Arbitrary TS.ShortText where
-  arbitrary = fmap TS.fromText arbitrary
-  shrink = map TS.fromText . shrink . TS.toText
-
```

Comment:
Arbitrary instance for `ShortText` already provided by `quickcheck-instances`, this is the reason I increased the lower bound on that dep.